docs/user/authoring-input-modes.rst

```diff
@@ -0,0 +1,69 @@
+Authoring Input Modes
+=====================
+
+# Overview
+
+A number of components provide multiple authoring modes.
+
+As a concrete example, the waves generated by the ShapeFFT component can be added to the world in multiple ways, including:
+
+* **Global** - waves simply added everywhere in the world
+* **Painted** - waves are painted down by hand in particular areas
+* **Spline** - waves are added along a user defined spline
+
+These are selected using the **Input Mode** dropdown on the ShapeFFT component, which display the valid modes for each component.
+
+The following sections describe each mode.
+
+
+# Global Mode
+
+The input simply applies everywhere.
+
+
+# Painted Mode
+
+The painting modality allows the input to be directly hand authored into the world. This is flexible and powerful. Keep in mind however that this data may need modification or re-authoring if the environment changes. It would be advisable to add quick input until the environment is locked down.
+
+Selecting Painted from the Mode dropdown will enable the Paint Mode Settings UI in the Inspector and make the painting tool available which can be enabled by either clicking the Crest Input Painting icon in the tool list, or by clicking Start Painting in the inspector.
+
+Once painting is enabled, a cursor is displayed on the water surface at the mouse position in the Scene view. We recommend reviewing the settings described below before painting in final data.
+
+By left click dragging the mouse, input can be added. For directional input types such as flow or waves, the direction of drag will specify the direction added to the data. On the other hand inputs like foam will simply add foam under the cursor. To remove data, hold shift while left click dragging.
```

Comment:
"By left click dragging the mouse" a little awkward. "left click and dragging" is easier to read.

Reply:
ah yes thanks